TESTING-CHECKLIST.md

```diff
@@ -0,0 +1,81 @@
+## Testing
+
+### Login
+
+- [ ] Logout
+- [ ] Login with wrong password fails
+- [ ] Login with correct password
+
+### Sync
+
+- [ ] Create new note appears in other device
+- [ ] Changes to new note sync to other device
+- [ ] Changes to new note sync _from_ other device
+- [ ] New tag immediately syncs to other device
+- [ ] New tag immediately syncs _from_ other device
+- [ ] Removed tag immediately syncs to other device
+- [ ] Removed tag immediately syncs _from_ other device
+- [ ] Note publishes with link
+- [ ] Note unpublishes
+- [ ] Note publish change syncs _from_ other device (visible with dialog open)
+- [ ] Markdown setting syncs _from_ other device
+- [ ] Preview mode disappears/reappears when receiving remote changes to markdown setting
+- [ ] Note pinning syncs immediately from either direction
+- [ ] Note pinning works regardless if clicking in list view or from note info (and syncs after the editor is opened or the app is resumed from being in the background)
+- [ ] Viewing history on one device leaves note unchanged on other device
+- [ ] Restoring history immediately syncs note from both directions
+- [ ] After disabling network connectivity and making changes, selecting Log Out triggers an `Unsynced Notes Detected` alert
+- [ ] After going back online, changes sync
+
+### Note editor
+
+- [ ] Can preview markdown with Preview tab (👁 button on landscape tablet)
+- [ ] Can flip to edit mode with Edit tab (👁 button on landscape tablet)
+- [ ] Can toggle sidebar (in landscape tablet)
+- [ ] Tapping the `Insert checklist` icon inserts a checklist
+- [ ] Typing `- [x]` creates a checked checklist item
+- [ ] Typing `- [ ]` created an unchecked checklist item
+- [ ] Typing `-` creates a list
+- [ ] All list bullet types render to markdown lists
+- [ ] Added URL is linkified
```

Comment:
```suggestion
- [ ] Added phone numbers and URL's are linkified
```

Phone numbers are also linkified.  Should we include that or is it too detailed?

Reply:
My gut says that testing a URL is probably sufficient, unless linkified phone numbers are both popular and can break separately from linkified URLs.

Reply:
I'm not sure how popular phone numbers are, but it shouldn't break separately from the URL's.  There are actually four variations of the linkified app bar.  See the screenshots below for illustration.

![linkified_variations](https://user-images.githubusercontent.com/3827611/82064994-bb490700-968a-11ea-9a78-6faa4d555691.png)

Testing any one of them should be sufficient to show linkification is working.

Reply:
Perfect, thanks for that comparison! Let's leave just the URL test in the checklist, to test the linkification. I'll make a note of these variations to keep it in mind for other testing.